src/plot_styles.py

```diff
@@ -253,105 +256,51 @@ def load_style(self):
             self.line_colors_box.append(box)
             self.color_boxes[box] = self.line_colors_box.get_last_child()
 
-        # other
-        self.axis_width.set_value(float(style["axes.linewidth"]))
-
     def save_style(self):
-        style = self.style
+        new_values = {key: None for key in STYLE_DICT.keys()}
+        ui.save_values_to_dict(self, new_values)
+        for key, value in new_values.items():
+            if value is not None:
+                for item in STYLE_DICT[key]:
+                    self.style[item] = value
 
         # font
         font_description = self.font_chooser.get_font_desc()
-        style["font.sans-serif"] = font_description.get_family()
+        self.style["font.sans-serif"] = font_description.get_family()
         font_name = font_description.to_string().lower().split(" ")
-        style["font.style"] = utilities.get_font_style(font_name)
+        self.style["font.style"] = utilities.get_font_style(font_name)
         font_weight = utilities.get_font_weight(font_name)
-        style["font.weight"] = font_weight
-        style["axes.titleweight"] = font_weight
-        style["axes.labelweight"] = font_weight
-        style["figure.titleweight"] = font_weight
-        style["figure.labelweight"] = font_weight
+        for key in ["font.weight", "axes.titleweight", "axes.labelweight",
+                    "figure.titleweight", "figure.labelweight"]:
+            self.style[key] = font_weight
         font_size = font_name[-1]
-        style["font.size"] = font_size
-        style["axes.labelsize"] = font_size
-        style["xtick.labelsize"] = font_size
-        style["ytick.labelsize"] = font_size
-        style["axes.titlesize"] = font_size
-        style["legend.fontsize"] = font_size
-        style["figure.titlesize"] = font_size
-        style["figure.labelsize"] = font_size
+        for key in ["font.size", "axes.labelsize", "xtick.labelsize",
+                    "ytick.labelsize", "axes.titlesize", "legend.fontsize",
+                    "figure.titlesize", "figure.labelsize"]:
+            self.style[key] = font_size
```

Comment:
Not really that different from the previous logic, but definitely more readable. Was told the other day something along the lines of "when you're declaring a whole load of similar stuff like that, you basically know you should be using some loop or comprehension". Which kinda applies to this.